src/Zadatak3/Complex.java

```diff
@@ -0,0 +1,55 @@
+package Zadatak3;
+
+public class Complex {
+    //Attributes
+    private int realPart;
+    private int imaginaryPart;
+
+    //Constructors
+    public Complex(){
+        this.realPart = 0;
+        this.imaginaryPart = 0;
+    }
+    public Complex(final int realPart, final int imaginaryPart){
+        this.realPart = realPart;
+        this.imaginaryPart = imaginaryPart;
+    }
+
+    //Getters
+    public int Re(){
+        return this.realPart;
+    }
+    public int Im(){
+        return this.imaginaryPart;
+    }
+
+    //Methods
+    public Complex add(Complex z){
+         int zOfRe = z.Re();
+         int zOfIm = z.Im();
+         Complex adding = new Complex(this.realPart+zOfRe, this.imaginaryPart+zOfIm);
+         return adding;
```

Comment:
no need for additional variables, just:

`return new Complex(this.realPart+ z.Re(), this.imaginaryPart+z.Im());`

Like for using z.Re() and this.realPart, since it emphasises the difference between parameter z and current object this.

Also you are the first person to call the parameter `z` instead of `z1` like for that :D.

Reply:
I get it. Fixed